azure-pipelines.yml

```diff
@@ -162,6 +162,52 @@ jobs:
               /p:TeamName=DotNetCore
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
             displayName: Sign packages
+    - job: Run_SDL
+      displayName: Run SDL tool
+      variables:
+        - group: SDL_Eng_KeyVault
+      steps:
+        - checkout: self
+          clean: true
+        - powershell: eng\common\build.ps1
+            -configuration Release
+            -restore
+            -prepareMachine
+            -ci
+            /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+          displayName: Windows Build / Publish
+        - task: NuGetToolInstaller@1
+          displayName: 'Install NuGet.exe'
+        - task: NuGetCommand@2
+          displayName: 'Install Guardian'
+          inputs:
+            restoreSolution: $(Build.SourcesDirectory)\eng\common\sdl\packages.config
+            feedsToUse: config
+            nugetConfigPath: $(Build.SourcesDirectory)\eng\common\sdl\NuGet.config
+            externalFeedCredentials: GuardianConnect
+            restoreDirectory: $(Build.SourcesDirectory)\.packages
+        # Script to execute SDL tools.
+        # TsaNotificationEmail will be changed to dnceng@
+        # to check what the noise level would be with regular SDL runs.
+        - powershell: eng/common/sdl/execute-all-sdl-tools.ps1
+            -GuardianPackageName Microsoft.Guardian.Cli.0.3.2
```

Comment:
Not blocking the PR on this, but seeing all the parameters that need to be passed, is every repo that onboards to this supposed to add all these values as part of their build yaml? Do you think there is some value in making this one of the templates that comes with Arcade?

Reply:
Yes every repo needs to add these params to get the SDL scripts added to their build/release. Adding the params to a template in Arcade is a good idea.